sdk/src/main/java/com/mapbox/maps/Style.kt

```diff
@@ -764,14 +834,23 @@ class Style internal constructor(
    * @return TRUE if and only if the style JSON contents, the style specified sprite and sources are all loaded, otherwise returns FALSE.
    */
   override fun isStyleLoaded(): Boolean {
-    return getStyleManager().isStyleLoaded
+    checkNativeStyle("isStyleLoaded")
+    return styleManager.isStyleLoaded
+  }
+
+  private fun checkNativeStyle(methodName: String) {
```

Comment:
what about getting `methodName` from stack trace?
only one concern that it can speed down a little bit code.

btw, maybe mark this function as `inline`?

Reply:
`inline` does not take effect if not using functional parameter types (that's what IDE told me when I tried to apply). Getting method from stacktrace may indeed be unexpectedly slow so guess we should not risk here